added namespace for AGP8 compatibility (#43)
Baseflow/flutter-google-api-availability: google_api_availability_android/CHANGELOG.md

## 1.0.1

> * Add support of namespace property to support Android Gradle Plugin (AGP) 8

Reviewer: nit: entries in the changelog should use present tense indicative for verbs, with "this version" as an implied subject. For example, "Adds cool new feature.", not "Add" or "Added".
```suggestion
* Adds support for the namespace property to support Android Gradle Plugin (AGP) 8.
```
